RecoEgamma/Configuration/python/RecoEgamma_EventContent_cff.py

```diff
@@ -192,11 +71,67 @@
+                                                                   'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerpp_*_*',
+                                                                   'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppIsland_*_*',
+                                                                   'keep recoPhotonCores_islandPhotonCore_*_*',
+                                                                   'keep recoPhotons_islandPhotons_*_*'])
+               )
+# RECO content
+RecoEgammaRECO = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'keep *_gedPhotonCore_*_*',
+        'keep *_gedPhotons_*_*',
+        #'keep *_gedPhotonsTmp_*_*',        
+        'keep recoPhotons_mustachePhotons_*_*',
+        'keep recoPhotonCores_mustachePhotonCore_*_*',
+        #'drop *_gedPhotonsTmp_valMapPFEgammaCandToPhoton_*',
+        'keep recoTrackExtras_ckfOutInTracksFromConversions_*_*', 
+        'keep recoTrackExtras_ckfInOutTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_ckfOutInTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_ckfInOutTracksFromConversions_*_*',
+        'keep recoConversions_uncleanedOnlyAllConversions_*_*',
+        'keep recoTrackExtras_uncleanedOnlyCkfOutInTracksFromConversions_*_*', 
+        'keep recoTrackExtras_uncleanedOnlyCkfInOutTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_uncleanedOnlyCkfOutInTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_uncleanedOnlyCkfInOutTracksFromConversions_*_*',
+        'keep recoRecoEcalCandidates_hfRecoEcalCandidate_*_*')                                                                 
+)
+RecoEgammaRECO.outputCommands.extend(RecoEgammaAOD.outputCommands)
+
+# Full Event content 
+RecoEgammaFEVT = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'keep *_gsfElectronCores_*_*', 
+        'keep *_gsfElectrons_*_*', 
+        'keep *_uncleanedOnlyGsfElectronCores_*_*', 
+        'keep *_uncleanedOnlyGsfElectrons_*_*', 
+        'keep *_eidRobustLoose_*_*',
+        'keep *_eidRobustTight_*_*',
+        'keep *_eidRobustHighEnergy_*_*',
+        'keep *_eidLoose_*_*',
+        'keep *_eidTight_*_*',
+        'keep *_egmGedGsfElectronPF*Isolation_*_*',
+        'keep *_egmGsfElectronIDs_*_*', 
+        'keep *_egmPhotonIDs_*_*',
+        'keep *_conversions_*_*',
+        'keep *_mustacheConversions_*_*',
+        'keep *_gedPhotonCore_*_*',
+        #'keep *_gedPhotonsTmp_*_*', #not really used
+        'keep *_gedPhotons_*_*',
+        'keep *_photonCore_*_*',
+        'keep *_photons_*_*',
+        'keep *_mustachePhotonCore_*_*',
+        'keep *_mustachePhotons_*_*',
+        'keep *_ootPhotonCore_*_*',
+        'keep *_ootPhotons_*_*',
+        'keep *_allConversions_*_*',
+        'keep *_allConversionsOldEG_*_*',
+        'keep *_ckfOutInTracksFrom*Conversions_*_*', 
```

Comment:
Is it possible to get rid of wild cards (here and in the line below)?

Reply:
There is only ckfOutInTracksFromConversions module in every event contents.
also for ckfInOutTracksFromConversions. So no need to use wild cards here.